db/docker-entrypoint.sh

```diff
@@ -2,7 +2,7 @@
 # Edited from https://github.com/arangodb/arangodb-docker/blob/official/alpine/3.6.4/docker-entrypoint.sh
 set -e
 
-if [ "$INIT_BRIGHTID_DB" == "1" ] || [ ! -f /var/lib/arangodb3/db_initialized ]; then
+if [ "$INIT_BRIGHTID_DB" == "1" ] || [ ! -f /var/lib/arangodb3/ENGINE ]; then
```

Comment:
What's the `ENGINE` file?

Reply:
It's one of the files that arangod saves on `data` named volume when the volume is created and db arangod runs for the  first time.

```
$ docker exec -it brightid-node_db_1 /bin/sh
/ # ls /var/lib/arangodb3
ENGINE          LANGUAGE        LOCK            SERVER          VERSION-1       engine-rocksdb
/ # 
```
I understand that there is a risk of having this file missed when we upgrade our arangodb version that can result in re-initialization of nodes but I did not find a better way to find out if `db` is initialized before or not. If we want to use this approach, we should be careful when upgrading our arango version.